### ISSUE TITLE
[PVC] Different torchbench behaviors observed on different PVC

### ISSUE THREAD

**Issue by kaileiyx** [intel/torch-xpu-ops#5395]
## Summary
2 torchbench models got a failure on PVC 1550, but passed on PVC 1100
## Affected Cases
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/kaileiyx/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/kaileiyx/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">

</head>

<body link="#467886" vlink="#96607D">

Suite | Dtype | Mode | Name | Target | Message
-- | -- | -- | -- | -- | --
torchbench | amp_fp16 | inference | hf_GPT2_large | eager_two_runs_differ | E0914 10:19:29.339000 270652 site-packages/torch/_dynamo/utils.py:3727]   Accuracy failed for key name logits
torchbench | bfloat16 | inference | sam | fail_accuracy | E0914 10:10:31.460000 115176 site-packages/torch/_dynamo/utils.py:3727]   Accuracy failed for key name masks
torchbench | float16 | inference | hf_GPT2_large | eager_two_runs_differ | E0914 10:11:51.416000 155471 site-packages/torch/_dynamo/utils.py:3727]   Accuracy failed for key name logits

</body>

</html>

## Reproducer
```bash
docker run -it -e TZ=Asia/Shanghai \
  --device=/dev/mem --device=/dev/dri --group-add video \
  --privileged --shm-size=8g \
  intelgpu/ubuntu-24.04-lts2:2523.40 bash

curl -LsSf https://astral.sh/uv/install.sh | env UV_INSTALL_DIR="/usr/local/bin" sh
uv venv myvenv --python 3.10 --clear
source myvenv/bin/activate
uv pip install pip wheel "setuptools<81"

uv pip install torch==2.15.0.dev20260912+xpu torchvision==0.30.0.dev20260913+xpu torchaudio==2.11.0.dev20260913+xpu triton-xpu==3.8.0+git1e2d42a0 --pre --index-url https://download.pytorch.org/whl/nightly/xpu

pytorch_commit="$(python -c 'import torch; print(torch.version.git_version)')"
git clone https://github.com/pytorch/pytorch
cd pytorch
git checkout ${pytorch_commit}

uv pip install pandas psutil scipy requests
uv pip install -U numpy==1.26.4 accelerate
uv pip install -r .ci/docker/ci_commit_pins/huggingface-requirements.txt
uv pip install timm

##benchmark
TORCHBENCH_COMMIT_ID="$(cat .ci/docker/ci_commit_pins/torchbench.txt 2> /dev/null || echo 'e03a63be')"
cd ..
git clone https://github.com/pytorch/benchmark benchmark
cd benchmark
python install.py --continue_on_fail hf_GPT2_large sam

pip install torchao==0.19.0.dev20260913+xpu --pre --index-url https://download.pytorch.org/whl/nightly/xpu
```
### PVC 1550
```bash
(myvenv) root@27e239f55931:/workspace/pytorch# python benchmarks/dynamo/torchbench.py --accuracy -d xpu -n10 --backend=inductor --cold-start-latency --inference --bfloat16 --only sam
loading model: 0it [00:07, ?it/s]
xpu  eval  sam
W0916 14:05:09.940000 20438 torch/_inductor/utils.py:3668] [0/0] get_device_tflops: no Triton fallback available for non-CUDA devices. Returning 0.0; roofline estimates will use memory bandwidth only.
W0916 14:05:37.269000 20438 torch/_inductor/cudagraph_utils.py:403] [0/0] [__cudagraphs] skipping cudagraphs due to skipping cudagraphs due to multiple devices: device(type='xpu', index=0)
E0916 14:05:46.748000 20438 torch/_dynamo/utils.py:3772] IoU accuracy failed: 0.0000 < 0.95 (intersection=0, union=1, ref_sum=1, res_sum=0, shape=[1, 1, 256, 256])
E0916 14:05:46.748000 20438 torch/_dynamo/utils.py:3727] Accuracy failed for key name masks
W0916 14:05:50.721000 20438 torch/_inductor/cudagraph_utils.py:403] [0/0] [__cudagraphs] skipping cudagraphs due to skipping cudagraphs due to multiple devices: device(type='xpu', index=0)
E0916 14:05:51.258000 20438 torch/_dynamo/utils.py:3772] IoU accuracy failed: 0.0000 < 0.95 (intersection=0, union=1, ref_sum=1, res_sum=0, shape=[1, 1, 256, 256])
E0916 14:05:51.258000 20438 torch/_dynamo/utils.py:3727] Accuracy failed for key name masks
W0916 14:05:55.199000 20438 torch/_inductor/cudagraph_utils.py:403] [0/0] [__cudagraphs] skipping cudagraphs due to skipping cudagraphs due to multiple devices: device(type='xpu', index=0)
E0916 14:05:56.303000 20438 torch/_dynamo/utils.py:3772] IoU accuracy failed: 0.0000 < 0.95 (intersection=0, union=1, ref_sum=1, res_sum=0, shape=[1, 1, 256, 256])
E0916 14:05:56.304000 20438 torch/_dynamo/utils.py:3727] Accuracy failed for key name masks
W0916 14:06:00.230000 20438 torch/_inductor/cudagraph_utils.py:403] [0/0] [__cudagraphs] skipping cudagraphs due to skipping cudagraphs due to multiple devices: device(type='xpu', index=0)
E0916 14:06:00.770000 20438 torch/_dynamo/utils.py:3772] IoU accuracy failed: 0.0000 < 0.95 (intersection=0, union=1, ref_sum=1, res_sum=0, shape=[1, 1, 256, 256])
E0916 14:06:00.770000 20438 torch/_dynamo/utils.py:3727] Accuracy failed for key name masks
W0916 14:06:04.667000 20438 torch/_inductor/cudagraph_utils.py:403] [0/0] [__cudagraphs] skipping cudagraphs due to skipping cudagraphs due to multiple devices: device(type='xpu', index=0)
E0916 14:06:05.788000 20438 torch/_dynamo/utils.py:3772] IoU accuracy failed: 0.0000 < 0.95 (intersection=0, union=1, ref_sum=1, res_sum=0, shape=[1, 1, 256, 256])
E0916 14:06:05.788000 20438 torch/_dynamo/utils.py:3727] Accuracy failed for key name masks
fail_accuracy

(myvenv) root@27e239f55931:/workspace/pytorch# clinfo | grep "Device Name"
  Device Name                                     Intel(R) Data Center GPU Max 1550
  Device Name                                     Intel(R) Data Center GPU Max 1550
    Device Name                                   Intel(R) Data Center GPU Max 1550
    Device Name                                   Intel(R) Data Center GPU Max 1550
    Device Name                                   Intel(R) Data Center GPU Max 1550
```

### PVC 1100
```bash
(myvenv) root@081899a9f582:/workspace/pytorch#  python benchmarks/dynamo/torchbench.py --accuracy -d xpu -n10 --backend=inductor --cold-start-latency --inference --bfloat16 --only sam
loading model: 0it [00:07, ?it/s]
xpu  eval  sam
W0916 14:10:18.817000 3449 torch/_inductor/cudagraph_utils.py:403] [0/0] [__cudagraphs] skipping cudagraphs due to skipping cudagraphs due to multiple devices: device(type='xpu', index=0)
W0916 14:10:35.983000 3449 torch/_inductor/cudagraph_utils.py:403] [0/0] [__cudagraphs] skipping cudagraphs due to skipping cudagraphs due to multiple devices: device(type='xpu', index=0)
W0916 14:10:39.704000 3449 torch/_inductor/cudagraph_utils.py:403] [0/0] [__cudagraphs] skipping cudagraphs due to skipping cudagraphs due to multiple devices: device(type='xpu', index=0)
W0916 14:10:44.000000 3449 torch/_inductor/cudagraph_utils.py:403] [0/0] [__cudagraphs] skipping cudagraphs due to skipping cudagraphs due to multiple devices: device(type='xpu', index=0)
W0916 14:10:47.706000 3449 torch/_inductor/cudagraph_utils.py:403] [0/0] [__cudagraphs] skipping cudagraphs due to skipping cudagraphs due to multiple devices: device(type='xpu', index=0)
pass

(myvenv) root@081899a9f582:/workspace/pytorch# clinfo | grep "Device Name"
  Device Name                                     Intel(R) Data Center GPU Max 1100
  Device Name                                     Intel(R) Data Center GPU Max 1100
  Device Name                                     Intel(R) Data Center GPU Max 1100
  Device Name                                     Intel(R) Data Center GPU Max 1100
    Device Name                                   Intel(R) Data Center GPU Max 1100
    Device Name                                   Intel(R) Data Center GPU Max 1100
    Device Name                                   Intel(R) Data Center GPU Max 1100
```

### Versions

Pytorch: nightly wheels
